koschei/backend/services/resolver.py

```diff
@@ -464,12 +464,12 @@ def generate_repo(self, sack, collection, repo_id):
             .filter_by(collection_id=collection.id)\
             .delete()
         prev_state = collection.state_string
+        collection.latest_repo_id = repo_id
```

Comment:
Can you describe the actual problem this is causing, or better, provide test case?

What's actually the meaning of `latest_repo_resolved`? I understand it as "build group in this repo is resolvable, feel free to submit builds". I don't see any problem there.